hasher-matcher-actioner/webapp/src/forms/PreviewableDropzone.tsx

```diff
@@ -84,17 +86,23 @@ export default function PreviewableDropzone({
 
   const {getRootProps, getInputProps, open, isDragActive} = useDropzone({
     onDrop,
-    accept: type === 'video' ? 'video/*' : 'image/*',
+    accept: type === ContentType.Video ? 'video/*' : 'image/*',
   });
 
+  const fileType = type === ContentType.Video ? 'video' : 'photo';
```

Comment:
Ignorable: why does `fileType` need to exist if it is 1-1 with ContentType, just use ContentType in PreviewPane?